oneadapter/src/main/java/com/idanatz/oneadapter/external/holders/ExternalHolders.kt

```diff
@@ -1,15 +1,15 @@
-package com.idanatz.oneadapter.internal.holders
+package com.idanatz.oneadapter.external.holders
 
 import com.idanatz.oneadapter.external.interfaces.Diffable
 import java.util.*
 
-internal abstract class InternalHolderModel : Diffable {
+abstract class ExternalHolderModel : Diffable {
```

Comment:
External has no clear meaning... 
If these classes should be exposed to the user than change the base class name from InternalHolderModel to OneHolderModel as its a general public holder model of OneAdapter

Reply:
Makes sense, il change it to OneHolderModel then. should I keep the new folder structure or put it back in internal/holders?